src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -83,15 +165,36 @@ export const ChannelView: React.FC = () => {
             )}
           </TitleContainer>
           <StyledButtonContainer>
-            <Button variant={isFollowing ? 'secondary' : 'primary'} onClick={handleFollow} size="large">
+            <StyledButton
```

Comment:
Seems this is missing the confirmation from the designs? Can be no problem done in a separate PR, just want to make sure we don't miss that. https://www.figma.com/file/TGUpBXgXf9ceMvHbRB5BIx/Joystream-Atlas?node-id=8171%3A210947

Reply:
Sure. In cases like this, when followup is needed, we should make sure there is an issue in the milestone tracking this required work